### ISSUE TITLE
Compilation error: SoftwareSerial.h: No such file or directory

### ISSUE THREAD

**Issue by Dopplerdave777** [flyandi/SquidRID#9]
I have tried several different versions of IDE and continue getting this error when compiling or uploading. I am also using an Sparkfun ESP32 board with the proper drivers and still getting this error.

**Comment by zeromonad** [comment]
Have you downloaded and installed the library?
![image](https://github.com/flyandi/SquidRID/assets/24425249/f8163d29-db8f-4850-9748-e4a12acff9d1)

**Comment by truglodite** [comment]
I had the same issue when I was using an older version of Arduino than the recommended 2.0.9. To fix it, after installing 2.0.9, I had to rename my old hardware folder, then install the esp32 board manager. After that it compiles fine, no ESPSoftwareSerial libraries needed (actually no additional libraries were needed at all after this).

FWIW, I did try that ESPSoftwareSerial library with the older IDE version and it still didn't compile.
